src/sentry/templates/sentry/emails/reports/body.html

```diff
@@ -229,7 +229,7 @@ <h1>
     {% with height=110 %}
     <table class="project-breakdown-graph-deck"><tbody><tr>
 
-    <td class="project-breakdown-graph-cell {% if has_replay_section %}errors-wide{% else %}errors{% endif %}" {% if has_replay_section %}colspan="2"{% endif %}>
```

Comment:
is `has_reply_section` still in use?

Reply:
the feature will be, the var not so much.

it'll be used once the query lands, wip here: https://github.com/getsentry/sentry/pull/51830/files

Reply:
I've removed that var from the template, and stubbed out `def key_replays()`  which is where the check will be needed.